src/pages/cis/[cisId].tsx

```diff
@@ -20,34 +23,46 @@ const SectionThree = () => (
   </div>
 );
 
-const PageSpeciality = () => (
-  <NavPageLayout
-    colorMenu="primary"
-    sections={[
-      {
-        id: 'data-ansm-question',
-        label: "DATA.ANSM c'est quoi ?",
-        content: <SectionOneGlobalInformation />,
-      },
-      {
-        id: 'donnees-globales-plateforme',
-        label: 'Données globales de la plateforme',
-        content: <SectionTwo />,
-      },
-      {
-        id: 'lecture-des-donnees',
-        label: 'Lecture des données',
-        content: <SectionThree />,
-      },
-    ]}
-    render={(content) => (
-      <>
-        <div>beforeContent</div>
-        {content}
-        <div>afterContent</div>
-      </>
-    )}
-  />
-);
+const PageSpeciality = () => {
+  const { subNavBarHeight } = useNavBarContext();
+  return (
+    <div>
+      <HeroSticky
+        tooltip={<CisTooltip />}
+        icon={<CisSvg className="h-fit" />}
+        title="Paracetamol eg 500 mg, gélule"
+        description="Spécialité de médicament"
+      />
+      <NavPageLayout
+        offsetHeight={subNavBarHeight}
+        colorMenu="primary"
```

Comment:
je me demande si cet `offsetHeight` peut pas carrément être posé ici avec un div paddingTop autour du `NavPageLayout` ??

Reply:
non mieux en fait @ruizalexandre 

en fait je pense que ton compo `HeroSticky`doit etre carrément embarqué dans le `NavPageLayout`, you see what I mean ??

Reply:
étant donné que ca fait partie du layout de ces 2 pages (spécialités et substances) ca a tout son sens !